src/libs/actions/IOU.ts

```diff
@@ -898,13 +898,6 @@ function buildOnyxDataForMoneyRequest(
         });
     }
 
-    // Show field violations only for control policies
```

Comment:
This just removed optimistic report violations, and doesn't really fix the bug at hand.

Reply:
Moving this into the issue https://github.com/Expensify/App/issues/47251